src/client/ws/manager.rs

```diff
@@ -38,7 +38,7 @@ type RequestId = u64;
 /// Manages and monitors JSONRPC v2 method calls and subscriptions.
 pub struct RequestManager {
 	/// List of requests that are waiting for a response from the server.
-	// NOTE: `FnvHashMap` is used here because `RequestId` is known to be short key (u64).
+	// NOTE: `FnvHashMap` is used here because `RequestId` is known to be a short key (u64).
```

Comment:
`// NOTE: `FnvHashMap` is used here because `RequestId` is not under the callers control and is known to be a short key (u64).`